### ISSUE TITLE
Bug with ACM bibtexfile

### ISSUE THREAD

**Issue by reinaqu** [IDEA-Research-Group/bibdiff#1]
There is a problem if we try to load the ACMDL201707122240630.bib file  (note that I have changed the extesion to .txt in order to upload the file to this issue)
[ACMDL201707122240630.bib.txt](https://github.com/IDEA-Research-Group/bibdiff/files/2776589/ACMDL201707122240630.bib.txt).

The error message was somethin like "KeyError: 'dec' bibtexparser py". 

After doing some research I have found this issue: https://github.com/sciunto-org/python-bibtexparser/issues/222, that seems to report on the same problem.

Following the recommendations I have modified the main function in the script,

```
def main():
    args = parse_arg()
    if not (os.path.isfile(args.bib[0]) and os.path.isfile(args.bib[1])):
        print("input file not found")
        exit(0)

    **parser = BibTexParser(common_strings=True)** 
    with open(args.bib[0]) as bibtex_file:
        database1 = bibtexparser.load(bibtex_file,**parser**)

    with open(args.bib[1]) as bibtex_file:
        database2 = bibtexparser.load(bibtex_file, **parser**)

    result = subtract(database1, database2)
    if args.output:
        with open(args.output, 'w') as bibtex_file:
            bibtexparser.dump(result, bibtex_file)
    else:
        print(bibtexparser.dumps(result))

```

I have enclosed the modifications in ** **

**Comment by jozemi** [comment]
Great! Thanks @reinaqu 

Can you please create Pull Request to master including that changes?

Thanks so much!

**Comment by jozemi** [comment]
The issue has been fixed in PR #2
